__tests__/app.test.js

```diff
@@ -239,10 +243,71 @@ test('400 status response with a non existent article id', () => {
       expect(body.msg).toEqual('Not found');
   }) 
 })
+
+describe('PATCH /api/articles/:article_id', () => { 
+  test('should return a status :200 and return a body with an updated increased number of votes', () => {
+    const newVote = 10;
+    const articleVoted = {inc_votes : newVote}         
+    return request(app)
+    .patch(`/api/articles/1`)
+    .send(articleVoted)
+    .expect(200)    
+    .then(({ body }) => {    
+      for (let i = 0; i < body.length; i++) {
+      expect(body[i].length).toEqual(1);      
+      expect(body[i].votes).toEqual(110);
+      }
+    });
+});
+test('should return a status :200 and return a body with an updated decreased number of votes', () => {
+  const newVote = -110;
+  const articleVoted = {inc_votes : newVote}         
+  return request(app)
+  .patch(`/api/articles/1`)
+  .send(articleVoted)
+  .expect(200)    
+  .then(({ body }) => {    
+    for (let i = 0; i < body.length; i++) {
```

Comment:
Same as above, this loop means the tests aren't actually happening, console.log the `body` and check what you're working with and what you need to test 🧪